tests/ceph_ansible/shrink_mon.py

```diff
@@ -37,14 +36,32 @@ def run(ceph_cluster, **kw):
     ceph_installer.node.obtain_root_permissions('/var/log')
     ansible_dir = ceph_installer.ansible_dir
 
-    ceph_installer.exec_command(sudo=True, cmd='cp -R {ansible_dir}/infrastructure-playbooks/shrink-mon.yml '
-                                               '{ansible_dir}/shrink-mon.yml'.format(ansible_dir=ansible_dir))
+    if build.startswith('4'):
```

Comment:
please add comments at each important step, so that the module readability will be easier.